internal/util/projutil/project_util.go

```diff
@@ -262,19 +261,15 @@ func IsOperatorAnsible() bool {
 }
 
 // IsOperatorHelm returns true when the layout field in PROJECT file has the Helm prefix key.
-// NOTE: For the legacy, returns true when the project contains the helm-charts directory.
 func IsOperatorHelm() bool {
-	// If the project has the new layout we will check the type in the config file
 	if kbutil.HasProjectFile() {
 		cfg, err := kbutil.ReadConfig()
 		if err != nil {
 			log.Fatalf("Error reading config: %v", err)
 		}
 		return PluginKeyToOperatorType(cfg.Layout) == OperatorTypeHelm
 	}
-	// todo(camilamacedo86): remove when the legacy layout is no longer supported
-	stat, err := os.Stat(helmChartsDir)
-	return (err == nil && stat.IsDir()) || os.IsExist(err)
+	return false
```

Comment:
Nit: I personally prefer the following.

```go
if !kbutil.HasProjectFile() {
	return false
}
cfg, err := kbutil.ReadConfig()
if err != nil {
	log.Fatalf("Error reading config: %v", err)
}
return PluginKeyToOperatorType(cfg.Layout) == OperatorTypeHelm
```

By including the HasFileProject check and immediate return early in the function, it makes it easier to understand more quickly that a Helm operator must have a PROJECT file to even be considered.

With the returns near the bottom of the function, I have to read more of the function to come to that understanding.

Reply:
Done 👍